### ISSUE TITLE
OCR Detection failed: The flag 'node_threshold' is defined twice

### ISSUE THREAD

**Issue by twist13227** [modelscope/modelscope#1101]
Hi! Seems like i have found a bug in OCRDetectionPipeline after the code update on November 1st.

**Summary**
A bug has been introduced in the OCRDetectionPipeline after merging commit fac865f. The node_threshold and link_threshold flags are now defined twice—both in the __init__() method (lines 90-93) and the preprocess() method (lines 202-205) of modelscope/pipelines/cv/ocr_detection_pipeline.py, resulting in a DuplicateFlagError.

**Bug details**
The following error is raised during execution:
absl.flags._exceptions.DuplicateFlagError: The flag 'node_threshold' is defined twice. First from modelscope.pipelines.cv.ocr_detection_pipeline, Second from modelscope.pipelines.cv.ocr_detection_pipeline.  Description from first occurrence: Confidence threshold for nodes

**To Reproduce**
Bug is reproduced even on slightly modified code from OCRDetectionPipeline documentation:
```python
    > from modelscope.pipelines import pipeline
    > from modelscope.utils.constant import Tasks
    > ocr_detection = pipeline(Tasks.ocr_detection, model='damo/cv_resnet18_ocr-detection-line-level_damo',)
    > result = ocr_detection('https://modelscope.oss-cn-beijing.aliyuncs.com/test/images/ocr_detection.jpg')
```
_Note_: The following code snippet in the documentation also contains a separate issue:
```python
    > ocr_detection = pipeline('ocr_detection', model='damo/cv_resnet18_ocr-detection-line-level_damo')
```
_This raises:_
KeyError: 'resnet18-ocr-detection is not in the pipelines registry group ocr_detection. Please make sure the correct version of ModelScope library is used.'). 
The `'ocr_detection'`  should be replaced with `Tasks.ocr_detection` to avoid this error.

**Environment Details**
* OS: Linux 6.8.0-48-generic #48-Ubuntu SMP PREEMPT_DYNAMIC Fri Sep 27 14:04:52 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
* Dependencies:
numpy>=1.22.3,<2.0.0
tqdm>=4.63.0,<5.0.0
psutil>=5.7.2,<6.0.0
tables==3.7.0
protobuf>=3.20.0,<4.0.0
scipy>=1.8.0,<1.9.0
torch==2.1.1
torchmetrics>=1.2.1,<2.0.0
torchvision==0.16.1
pytorch-lightning>=2.0.6,<3.0.0
modelscope>=1.16.0,<2.0.0
addict>=2.4.0,<3.0.0
datasets==2.18.0
oss2>=2.18.6,<3.0.0
simplejson>=3.19.2,<4.0.0
tf-slim>=1.1.0,<2.0.0
pyclipper>=1.3.0.post5,<2.0.0
shapely>=2.0.4,<3.0.0
sortedcontainers>=2.4.0,<3.0.0
tensorflow==2.12.1
tf-keras==2.15
typing-extensions==4.5.0
opencv-python-headless==4.5.4.60
* Commit id (e.g. `fac865f`)

**Proposed Fix**
To resolve the DuplicateFlagError, either:
- Remove the redundant flag definitions from the preprocess() method;
- Ensure the flags are defined only once during the pipeline initialization.

@tastelikefeet @wangxingjun778 Check it please, thank you in advance for your attention!

**Comment by twist13227** [comment]
I see it from line 42 of modelscope/pipelines/cv/ocr_detection_pipeline.py
You can reproduce this sub-bug if you launch example code from OCRDetectionPipeline docstring:
```python
    > from modelscope.pipelines import pipeline
    > ocr_detection = pipeline('ocr_detection', model='damo/cv_resnet18_ocr-detection-line-level_damo')
    > result = ocr_detection('https://modelscope.oss-cn-beijing.aliyuncs.com/test/images/ocr_detection.jpg')
```

Anyway, main bug is much more important. This one is just a slight inaccuracy in class documentation.

**Comment by yingdachen** [comment]
#1102 fixed

**Comment by twist13227** [comment]
I highly appreciate your work and will wait for package update in PyPI!
